public/javascripts/items.js

```diff
@@ -10,25 +10,83 @@ const fetchParams = {
 const params = completed =>
   Object.assign( {}, fetchParams, { body: JSON.stringify({ completed }) })
 
+const params2 = title =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ title }) })
+
+const params3 = description =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ description }) })
+
 $(document).ready( () => {
-  $( '.title' ).click( event => {
+  $('.updateTitle').keypress( event => {
+    element = $( event.target )
+    const id = element.data( 'id' )
+
+    if( event.charCode === 13 ) {
```

Comment:
I like to remove magic numbers and replace them with constants...  This one is OK since most programmers will understand the meaning of 13 here (it's a fairly common pattern in UI JS), but consider the following:
```js
const RETURN_KEY = 13

// later on:
if( event.charCode === RETURN_KEY ) {
  // etc.
}

```